spinta/backends/postgresql/constants.py

```diff
@@ -8,3 +8,6 @@
     'generic',
     'rql',
 ]
+
+WGS84 = 4326
+LKS94 = 3346
```

Comment:
This should go to `spinta.types.geometry.constants`, because these numbers has nothing to do with PostgreSQL it is specific to `geometry` type.